### PR TITLE
Add documentation for events data flow

### PR DESCRIPTION
**Pull request by georgf**
This adds an overview of how event data flows through our data pipeline end-to-end.

For now this is mostly aimed at what users need to know, so i can link
something proper in the newsletter.
I'd love to see more technical detail in this later, but i think this scope is sufficient for a first pass.

### CODE REVIEW

**Comment by georgf** [comment]
@mreid-moz, what do you think about adding a new sub-article like this to the pipeline documentation?
I feel this is too long and too specific to fit into the other articles.

**Comment by georgf** [comment]
Rendered diagram for convenience:

File: concepts/pipeline/event_pipeline.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+On the pipeline side, the event data is made available in different datasets:
+- [`main_summary`](/concepts/choosing_a_dataset.md#mainsummary) has a row for each main ping and includes
+  its event payload.
+- [`events`](/datasets/batch_view/events/reference.md) contains a row for each event received.
+- `focus_events_longitudinal` contains the current mobile events.
+
+# Data tooling
+
+The above datasets are all accessible through [Re:dash](/tools/stmo.md) and [Spark jobs](/tools/stmo.md).
```
Comment (mreid-moz) [inline concepts/pipeline/event_pipeline.md]: nit: Spark link is pointing to the wrong page

File: concepts/pipeline/event_pipeline.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+with a hard limit of 500 events per ping.
+In the future, Firefox events will be sent through a separate *events ping*, removing the hard limit.
+As of Firefox 61, all events recorded through these APIs are [automatically counted in scalars](https://bugzilla.mozilla.org/show_bug.cgi?id=1440673).
+
+Finally, [*custom pings*](https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/collection/custom-pings.html)
+can follow the event data format and potentially connect to the existing tooling with some integration work.
+
+## Mobile event collection
+
+On mobile, event data is currently collected through the [`focus-events` ping](https://github.com/mozilla-mobile/focus-ios/wiki/Event-Tracking-with-Mozilla%27s-Telemetry-Service#event-ping),
```
Comment (fbertsch) [inline concepts/pipeline/event_pipeline.md]: I would change this to the `mobile-events` ping, which is the one we want people to be using.

File: concepts/pipeline/event_pipeline.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+firefox -->|main ping| pipeline((fa:fa-database Firefox Data Pipeline))
+firefox -.->|events ping, planned| pipeline
+mobile_telemetry --> |mobile events ping| pipeline
+pipeline -->|Firefox events| main_summary[fa:fa-bars main summary table]
+main_summary --> events_table[fa:fa-bars events table]
+pipeline -->|Mobile events| focus_events_table[fa:fa-bars mobile events table]
+main_summary --> redash(fa:fa-bar-chart Redash)
+events_table --> redash
+events_table -.->|planned| amplitude(fa:fa-bar-chart Amplitude)
+focus_events_table --> redash
```
Comment (fbertsch) [inline concepts/pipeline/event_pipeline.md]: I would change this to `mobile_events`, here and everywhere else. While technically we are sending events to Amplitude from `focus-events`, we also don't want anyone else to use that ping, and will eventually transition focus to `mobile-events` as well.

File: concepts/pipeline/event_pipeline.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+[`telemetry-android`](https://github.com/mozilla-mobile/telemetry-android) libraries.
+In the future event data will be mainly collected through the planned mobile events ping.
+
+# Datasets
+
+On the pipeline side, the event data is made available in different datasets:
+- [`main_summary`](/concepts/choosing_a_dataset.md#mainsummary) has a row for each main ping and includes
+  its event payload.
+- [`events`](/datasets/batch_view/events/reference.md) contains a row for each event received.
+- `focus_events_longitudinal` contains the current mobile events.
```
Comment (fbertsch) [inline concepts/pipeline/event_pipeline.md]: This should be `telemetry_mobile_events_parquet`.

File: concepts/pipeline/event_pipeline.md [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+
+# Data tooling
+
+The above datasets are all accessible through [Re:dash](/tools/stmo.md) and [Spark jobs](/tools/spark.md).
+
+For product analytics based on event data, we have [Amplitude](https://sso.mozilla.com/amplitude)
+(hosted by the IT data team). We can connect our event data sources data to Amplitude.
+We have an active connector to Amplitude for mobile events, which can push event data over
+daily. For Firefox Desktop events this is planned and available soon.
```
Comment (fbertsch) [inline concepts/pipeline/event_pipeline.md]: nit: "will be available soon"

File: concepts/pipeline/event_pipeline.md (hunk 2 above)
Comment (georgf) [reply concepts/pipeline/event_pipeline.md]: Do we support this ping yet?
We probably should clearly call out what we have and what is just coming.

Comment (fbertsch) [reply concepts/pipeline/event_pipeline.md]: For over 4 months now :)

https://github.com/mozilla-services/mozilla-pipeline-schemas/tree/dev/schemas/telemetry/mobile-event

File: concepts/pipeline/event_pipeline.md [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+
+Across the different Firefox teams there is a common need for a more fine grained understanding of
+product usage, like understanding the order of interactions or how they occur over time.
+To address that our data pipeline needs to support working with event-oriented data.
+
+We specify a common event data format, which allows for broader, shared usage of data processing tools.
+To make working with event data feasible, we provide different mechanisms to get the event data
+from products to our data pipeline and make the data available in tools for analysis.
+
+# The event format
```
Comment (sunahsuh) [inline concepts/pipeline/event_pipeline.md]: Should we provide a link to the corresponding section in the source docs in this section as well?

**Pull request merged**